service-node/src/main/scala/org/enterprisedlt/fabric/service/node/process/DockerManagedBox.scala

```diff
@@ -80,8 +81,50 @@ class DockerManagedBox(
 
 
     //=========================================================================
+
+    def pullImageIfNeeded(image: String, forcePull: Boolean = false): Unit = {
+        val images = docker
```

Comment:
no need to search if pull is forced

Reply:
Not sure how to implement it in the current implementation.

Reply:
Done